ApplePaySession.js

```diff
@@ -385,6 +391,19 @@
 
             };
 
+            /**
+             * Callback for ApplePaySession.completePaymentMethodSelection() for Apple Pay JS version 3 and above.
+             * @param {Object} session - The current ApplePaySession.
+             * @param {Object} applePayPaymentAuthorizedEvent - The event sent to onpaymentauthorized.
+             */
+            self.onPaymentAuthorizedWithTimeout = function (session, applePayPaymentAuthorizedEvent) {
+                session.onpaymentauthorized(applePayPaymentAuthorizedEvent);
+
+                self.authorizationTimeout = setTimeout(() => {
+                    alert('ApplePay Not Finished - The site was not able to complete the payment. Please, trye again');
+                }, 30000);
```

Comment:
Would there be any value in making this configurable?

Reply:
This is hardcoded by Apple. But for sure would be nice to allow devs to change for debugging.